aiken/lib/cardano_loans/interest_validation.ak

```diff
@@ -156,7 +158,8 @@ fn process_input(
               min_payment > total_epoch_payments, // Penalize if min_payment not met.
               penalty,
               loan_outstanding,
-              loan_interest,
+              // Don't apply interest unless it is compounding interest.
+              if compounding_interest {loan_interest} else {(0,1)},
```

Comment:
The interest is now only applied if it is compounding interest. If it is none compounding, an interest of zero is passed in instead so that penalties can still be applied.